core/storage/statistics/gae_models.py

```diff
@@ -23,11 +23,29 @@
 import json
 import sys
 
+# TODO(#13594): After the domain layer is refactored to be independent of
+# the storage layer, the disable=invalid-import will
+# be removed.
+# The following import is dangerous and should not be generally
+# used. We had to use this ignore as we need to import the domain layer
+# for type-annotation.
+# Here exp_domain is imported outside the 'if MYPY:' block. If it is
+# imported inside the 'if MYPY:' block, during runtime exp_domain
+# won't be accessible.
+from core.domain import exp_domain # pylint: disable=invalid-import
```

Comment:
Flagging this for @vojtechjelinek to look at.

Reply:
Opening other thread so I can track it more easily.